src/Html.php

```diff
@@ -4813,7 +4813,8 @@ public static function jsAjaxDropdown($name, $field_id, $url, $params = [])
 
        // manage multiple select (with multiple values)
         if ($params['multiple']) {
-            $values = array_combine($params['values'], $params['valuesnames']);
+            $min = min(count($params['values']), count($params['valuesnames']));
```

Comment:
Isn't that a bug when `values` and `valuesnames` do not have the same number of elements? I wonder if the proposed fix won't just hide another issue.

Reply:
I agree, having a different number of elements here is an issue.